docs/Installation.md

```diff
@@ -1,70 +1,127 @@
+![IIHDatabusSettings](/docs/graphics/IIHDatabusSettings.png]
 
-### Configure Databus
+As soon as the SIMATIC S7+ Connector is installed and started on the same IED as the Common Configurator, the connector is visible within the configurator. In this example we want to configure a S7+ connection to a CPU 1515F-2. It is required to have an SIMATIC SCADA export of the dedicated TIA project available (Export.zip).
 
-In your IEM open the IE Databus and launch the configurator.
+- go to the tab 'Get data'
+- select the SIMATIC S7+ Connector
+- switch to tab 'Tags'
+- add a new data source
+- choose 'add from file'
+- select the Export.zip from the SIMATIC SCADA Export of the TIA project
+- add a name and the IP address of the PLC
+- select the option 'Provide PLC alarms'
+- click 'Continue to "Select tags"'
 
-Add a user with this topic:
-`"ie/#"`
+![DataSource](/docs/graphics/IIHDataSource.png)
 
-![databus](/docs/graphics/Databus.PNG)
+- select an acquisition cycle and a access mode in the drop down fields
+- select the parameter that represents the machine state
+- click 'Import'
+
+![DataSource2](/docs/graphics/IIHDataSource2.png)
+
+For writing the tag values onto the MQTT databus you need to activate and confirm the 'Publish on the databus' option for each tag:
+
+- select the parameter
+- click on the edit icon of the parameter
+- activate the option 'publish on the databus'
+- select an acquisition cycle 
+- accept the settings
+
+![DataSource3](/docs/graphics/IIHDataSource3.png)
+
+Select the newly created PLC including all the tags and click 'Deploy' to save the configuration and start the project.
+
+**Important to know:**
+The alarms can only be activated when creating the data source. It is not possible to add them afterwards!
+
+## Configure IIH Essentials
+
+Within **IIH Essentials** you can define the asset structure for your plant: 
+
+- open the app IIH Essentials
+- create the asset hirarchy accordingly
+
+The app gathers all the necessary process data and saves it for a configured period of time. In this case we want to get data out of the S7+ Connector, therefore this must be activated:
 
-Deploy the configuration.
+- go to the tab 'Settings' and select 'Databus settings'
+- enter the databus service name: `ie-databus:1883`
+- enter the user name and password ('edge'/'edge')
+- save
 
-### Configure OPC UA Connector
+- go to the tab 'Connectors'
+- select the 'SIMATIC S7 Plus Connector'
+- activate and save
 
-In your IEM open the OPC UAgit Connector and launch the configurator.
+![IIHEssentialsAdaper](/docs/graphics/IIHEssentialsAdapter.png)
 
-Add a data source:
+To add the PLC parameter for the machine state, proceed as following:
 
-![OPC UA Connector Data Source](/docs/graphics/OPCUA_Connector_Data_Source.PNG)
+- go to tab 'Assets & Connectivity'
+- select the dedicated asset
+- clickt 'Create first variable'
+- select the SIMATIC S7 Plus Connector
```

Comment:
Here change SIMATIC S7 plus Connector to SIMATIC S7+ Connecto

Reply:
In IIH Essentials it is named exactly like this 'SIMATIC S7 Plus Connector'.